### ISSUE TITLE
Improve documentation on PairingError: <TlvValue.Salt: 2>

### ISSUE THREAD

**Issue by IsakTheHacker** [postlund/pyatv#2294]
### What to change?

If I try to pair with a device but fail after multiple attempts (don't know exactly how many), pyatv throws:
```
PairingError: <TlvValue.Salt: 2>
```

My suggestion is to change this to something that is easier to understand, such as:
```
PairingError: Too many failed attempts, try again later!
```

**Comment by postlund** [comment]
This is a different error, there actually is a "backoff" error after too many attempts. What kind of device is this?

**Comment by IsakTheHacker** [comment]
Hmm, that's weird. I'm using a 4th generation Apple TV (not the 4K model).
One additional thing of note is that the pincode actually is displayed on
the screen, but I'm not able to perform pairing since the exception is
thrown when calling PairingHandler.begin() in the pyatv API.

**Comment by postlund** [comment]
Yes, now when I look at it, it looks a bit a strange. This suggests that the Apple TV does not include the salt value but also not any error. Can you run with `--debug` and include the logs?

**Comment by IsakTheHacker** [comment]
Where should I pass the `--debug` option? To atvremote?

My setup is quite involved so I don't know if the same error would appear when using the command line utilities. Is it possible to turn on debug logging when using the API?

**Comment by postlund** [comment]
Sure, should be something like `logging.getLogger("pyatv").setLevel(logging.DEBUG)`.
